dvol_python/dvol.py

```diff
@@ -166,9 +182,13 @@ def createBranch(self, volume, branch):
         self.output("Created branch %s/%s" % (volume, branch))
 
     def createVolume(self, name):
-        if self._directory.child(name).exists():
-            self.output("Error: volume %s already exists" % (name,))
-            raise VolumeAlreadyExists(name)
+        try:
+            if self._directory.child(name).exists():
+                self.output("Error: volume %s already exists" % (name,))
+                return
+        except InsecurePath:
+            self.output("Error: %s is not a valid name" % (name,))
```

Comment:
:question: This error message is bad UX, since it doesn't help the user figure out what a valid name is. 

Since the Python code is going away, and since we don't have a clear definition of names, don't need to fix this. But probably also don't force the Go code to have the same error.